integration-tests/regression.test.js

```diff
@@ -89,7 +94,7 @@ describe('Tram-One', () => {
 		});
 
 		// clear the input
-		userEvent.type(getByLabelText(container, 'New Task Label'), '{selectall}{backspace}');
+		userEvent.clear(getByLabelText(container, 'New Task Label'));
```

Comment:
didn't realize that this `clear` method existed, but yeah, it very specifically says it [selects the text and then deletes it](https://testing-library.com/docs/ecosystem-user-event/#clearelement)